core/capabilities/remote/executable/client.go

```diff
@@ -2,11 +2,12 @@ package executable
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"sync"
 	"time"
 
+	"github.com/pkg/errors"
```

Comment:
this is depreciated. pls use stdlib errors

Reply:
hmm I wanted to use Wrap() - is Join() a recommended substitute?

Reply:
changed to Join(), thanks